### DISCUSSION TITLE
Tree select styling missing on edge verion 80

### DISCUSSION THREAD

**Discussion by Mnehaal2000** [primefaces/community#3040] [PrimeReact]
I am using the treeselect component and it loses all its styling on older microsoft edge versions. I am looking at version 80 as that is what my client requires. Screenshots attached are of the component running in latest chrome and the older edge.

Link to the component i am using:https://primereact.org/treeselect/,
![Screenshot 2024-11-15 143500](https://github.com/user-attachments/assets/911393a6-0f10-406c-a815-60ccd5d36e82)
![Screenshot 2024-11-15 143334](https://github.com/user-attachments/assets/b26687e2-e516-472c-977e-8ceed2ff8a34)

**Comment by melloware** [comment]
i doubt older Edge will be supported.
